src/SignalR/clients/ts/signalr-protocol-msgpack/package.json

```diff
@@ -44,11 +39,10 @@
     "src/**/*"
   ],
   "dependencies": {
-    "@microsoft/signalr": "link:../signalr",
+    "@microsoft/signalr": "^5.0.0-dev",
```

Comment:
Same here

Reply:
[I've updated the packing process to go and override the dependencies
![image](https://github.com/dotnet/aspnetcore/assets/6995051/977be1a4-3f4b-4a1f-b70f-325224de1975)
](https://github.com/dotnet/aspnetcore/pull/52914#discussion_r1433185214)